src/main/resources/application.properties

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
+info.build.version=1.9
+
+rendezvous.hmacSecret=efe08a5df188dac44a4534c9f8ca0aed1b7ef6003622a07bf8a331134b3c1af427a6d7ee298f25d7f911ae495a8356e4636c19ada91643bd0d8b4f7b2bb97156
+rendezvous.tOTokenExpirationTime=15
+rendezvous.verificationServiceHost=https://verify.epid-sbx.trustedservices.intel.com:443
```

Comment:
In docker-compose file, the port information has been removed from the variable definition.

Reply:
Removed port number